queries.sql

```diff
@@ -22,3 +22,94 @@ WHERE name <> 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* New queries */
+-- First transaction
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+-- Second transaction
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+SELECT * FROM animals;
+
+COMMIT;
+
+SELECT * FROM animals;
+
+-- Third transaction
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+-- forth transaction
+
+BEGIN;
+
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT save_point_1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO save_point_1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+COMMIT;
+
+-- Answers
+
+-- How many animals are there?
+SELECT COUNT(*) AS total_animals
+FROM animals;
+
+--How many animals have never tried to escape?
```

Comment:
- [ ] Please kindly adhere to the project requirement its essential to add screenshots of their transation results to the pull request description. **You should check ** This also helps to ensure that students run the queries with success. should add screenshots of their transation results to the pull request description.